server/main.lua

```diff
@@ -25,5 +25,7 @@ local function setActivityBusy(name, bool)
     illegalActions[name].busy = bool
     GlobalState.illegalActions = illegalActions
 end
+
+---@deprecated use the setActivityBusy export instead
 RegisterNetEvent('qb-scoreboard:server:SetActivityBusy', setActivityBusy)
```

Comment:
Is this being used from the client currently? Should we just remove it instead?

Reply:
Several resources use this. It is also worth to pay attention to the compatibility with custom resources.
examples:
[qbx_bankrobbery](https://github.com/Qbox-project/qbx_bankrobbery/blob/6fe6695445df47fd7173d32c02d0e064a936cbc6/server/main.lua#L127)
[qbx_jewelery](https://github.com/Qbox-project/qbx_jewelery/blob/e01e68d15ee8b885456b0936700313677ca72114/server/main.lua#L80)

All resources in the full recipe use this event from the server side script.

Reply:
We don't have to keep compatibility for non core resources, but if you want to, it's fine as well.

Reply:
I would keep this until qbx resources are updated.